### DISCUSSION TITLE
Create local album from uploaded album

### DISCUSSION THREAD

**Discussion by HippoBloke** [immich-app/immich#6978] [Q&A] [answered]
Hi all

New user and loving it so far. I have uploaded album son my iPhone from Apple photos. It shows on the Library page as 'Album on Device.

I want to create a local album of the same name to view on other devices. This far I have been selecting photos from the device album and creating a new album. This is a little tedious as the photos have to be selected one by one after opening the device album, there is no select all I can find.

Am I missing something in creating albums this way or have I missed the point altogether ?

Thanks
Hippotech

**Comment by bo0tzz** [comment, marked as answer]
#1678

**Reply by HippoBloke** [reply]
Thanks - lots of variations but my workflow is in there too.
